rtsl_util/CommonUtils/Dhis2CucumberTestTool/src/test/resources/local_tests/scenarios/hypertension/hypertension_overdue_patients_called.feature

```diff
@@ -0,0 +1,121 @@
+      | Systole  | 142 |
+      | Diastole | 95  |
+    And That TEI has a "Calling report" event on "6_MonthsAgo_Plus_1_Day" with following data
+      | Result of call                          | AGREE_TO_VISIT |
+    And That TEI has a "Calling report" event on "3_MonthsAgo_Plus_1_Day" with following data
+      | Result of call                          | REMOVE_FROM_OVERDUE |
+      | HTN - Remove from overdue list because: | OTHER               |
+
+    When I export the analytics
+
+    Then The value of "PI":"HTN - Overdue patients called" with period type "Months" should be
+      | 3_MonthsAgo | 1 |
+      | 4_MonthsAgo | 0 |
+      | 5_MonthsAgo | 0 |
+      | 6_MonthsAgo | 1 |
+      | 7_MonthsAgo | 0 |
+
+  Scenario: Only managed patients should be counted (Non-hypertensive, alive, and in-facility)
+    Given I am signed in as a user with role "Superuser"
+    And I have access to an organisation unit at level 5
+    And I register that organisation unit for program "Hypertension & Diabetes"
+
+    # Create the patients
+    And I create a new TEI on "7_MonthsAgo" for this Facility with the following attributes
+      | GEN - Given name                      | Nonne        |
+      | GEN - Family name                     | Hypertensive |
+      | GEN - Sex                             | MALE         |
+      | HTN - Does patient have hypertension? | NO           |
+      | HTN - Does patient have diabetes?     | YES          |
+      | GEN - Date of birth                   | 32           |
+      | Address (current)                     | By the beach |
+      | District                              | KOLARA       |
+      | HTN - Consent to record data          | true         |
+      | HTN - NCD Patient Status              | ACTIVE       |
+    And That TEI has a "Hypertension & Diabetes visit" event on "7_MonthsAgo" with following data
+      | Systole  | 119 |
+      | Diastole | 69  |
+    And That TEI has a "Calling report" event on "6_MonthsAgo_Plus_1_Day" with following data
+      | Result of call                          | AGREE_TO_VISIT |
+    And I create a new TEI on "5_MonthsAgo" for this Facility with the following attributes
+      | GEN - Given name                      | Swimming with|
+      | GEN - Family name                     | The Fishes   |
+      | GEN - Sex                             | MALE         |
+      | HTN - Does patient have hypertension? | NO           |
+      | HTN - Does patient have diabetes?     | YES          |
+      | GEN - Date of birth                   | 32           |
+      | Address (current)                     | By the beach |
+      | District                              | KOLARA       |
+      | HTN - Consent to record data          | true         |
+      | HTN - NCD Patient Status              | ACTIVE       |
+    And That TEI was updated on "4_MonthsAgo" with the following attributes
```

Comment:
Let's also give a calling report to this patient. So that `status=DIED` becomes the only criteria to not include this patient.